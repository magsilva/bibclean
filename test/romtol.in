i
ii
iii
iiii
iv
v
vi
vii
viii
viiii
ix
x
xi
xii
xiii
xiiii
xiv
xv
xvi
xvii
xviii
xviiii
xix
xx
xxx
xxxx
xxxxx
l
c
cc
cd
d
dc
cm
m
mcm
mcmxcii
mxmii
abc
iabc
vabc
xabc
labc
cabc
dabc
mabc
