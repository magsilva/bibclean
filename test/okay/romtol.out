romtol():  i -> 1	Remainder = []
romtol():  ii -> 2	Remainder = []
romtol():  iii -> 3	Remainder = []
romtol():  iiii -> 4	Remainder = []
romtol():  iv -> 4	Remainder = []
romtol():  v -> 5	Remainder = []
romtol():  vi -> 6	Remainder = []
romtol():  vii -> 7	Remainder = []
romtol():  viii -> 8	Remainder = []
romtol():  viiii -> 9	Remainder = []
romtol():  ix -> 9	Remainder = []
romtol():  x -> 10	Remainder = []
romtol():  xi -> 11	Remainder = []
romtol():  xii -> 12	Remainder = []
romtol():  xiii -> 13	Remainder = []
romtol():  xiiii -> 14	Remainder = []
romtol():  xiv -> 14	Remainder = []
romtol():  xv -> 15	Remainder = []
romtol():  xvi -> 16	Remainder = []
romtol():  xvii -> 17	Remainder = []
romtol():  xviii -> 18	Remainder = []
romtol():  xviiii -> 19	Remainder = []
romtol():  xix -> 19	Remainder = []
romtol():  xx -> 20	Remainder = []
romtol():  xxx -> 30	Remainder = []
romtol():  xxxx -> 40	Remainder = []
romtol():  xxxxx -> 50	Remainder = []
romtol():  l -> 50	Remainder = []
romtol():  c -> 100	Remainder = []
romtol():  cc -> 200	Remainder = []
romtol():  cd -> 400	Remainder = []
romtol():  d -> 500	Remainder = []
romtol():  dc -> 600	Remainder = []
romtol():  cm -> 900	Remainder = []
romtol():  m -> 1000	Remainder = []
romtol():  mcm -> 1900	Remainder = []
romtol():  mcmxcii -> 1992	Remainder = []
romtol():  mxmii -> 1992	Remainder = []
romtol():  abc -> 0	Remainder = [abc]
romtol():  iabc -> 1	Remainder = [abc]
romtol():  vabc -> 5	Remainder = [abc]
romtol():  xabc -> 10	Remainder = [abc]
romtol():  labc -> 50	Remainder = [abc]
romtol():  cabc -> 100	Remainder = [abc]
romtol():  dabc -> 500	Remainder = [abc]
romtol():  mabc -> 1000	Remainder = [abc]
